Compilation-Alpine-Linux.md

```diff
@@ -34,10 +34,10 @@ apk add icu-dev openssl-dev qt6-qtbase-dev qt6-qttools-dev zlib-dev
 
 ## Boost build files
 
-> [!TIP]>
+> [!TIP]
 > This command should provide the latest non beta release info from Github
 >
-> - `curl -sL https://api.github.com/repos/boostorg/boost/releases | jq -r '.[0].name | select(contains("beta") | not)'`
+> - `curl -sL https://api.github.com/repos/boostorg/boost/releases | jq -r '.[].name | select(contains("beta") | not)' | head -n 1'`
```

Comment:
How about this?
```suggestion
> - `curl -sL https://api.github.com/repos/boostorg/boost/releases | jq -r 'map(.name | select(contains("beta") | not)) | first'`
```

Reply:
Also it would be more robust if you used regex to match, instead of just filtering out `beta`:
```suggestion
> - `curl -sL https://api.github.com/repos/boostorg/boost/releases | jq -r 'map(.name | select(test("boost-[\\d\\.]+$"))) | first'`
```